build.gradle

```diff
@@ -188,6 +188,7 @@ dependencies {
     compile 'org.liquibase:liquibase-core:3.8.0'         // For upgrade
     compile 'org.postgresql:postgresql:42.2.18'           // Postgres jdbc driver
     compile 'org.slf4j:slf4j-api:1.7.28'                 // Logging facade
+    compile 'org.codehaus.janino:janino:3.1.6'
```

Comment:
What does this library do?

Reply:
Hmm..I think it's a java compiler...not sure why we need to add it?  Maybe worth a comment?

Reply:
Provides the if/else capabilities in the logback XML to enable our local vs stackdriver logging. I can comment that in the code.

Reply:
ooooh.  cool.